src/pages/BusinessSpecifics/BusinessSpecifics.tsx

```diff
@@ -28,8 +49,9 @@ export const BusinessSpecifics = () => {
                 {currencyList}
             </TextField>
             <Button variant="contained"
-            disabled={!BusinessName || !LocationName || !currency }
-            onClick={() => console.log({BusinessName, LocationName, currency})}>submit</Button>
+            disabled={!businessName || !locationName || !currency }

```

Comment:
this logic is fine but we can avoid it by just adding validators in the form fields, we need to disable this button after the user successfully submits the form to prevent overfetching

Reply:
got it